maas/client/viscera/machines.py

```diff
@@ -67,11 +67,17 @@ class MachinesType(ObjectType):
         else:
             return cls._object(data)
 
-    async def get_power_parameters(
-            cls, *, system_ids: typing.Sequence[str]=[]):
+    async def get_power_parameters_for(
+            cls, *, system_ids: typing.Sequence[str]):
```

Comment:
Why does system_ids need to be a kwarg? Why force this? Why not just allow:

`get_power_parameters_for([...])`

Since its the only parameter that it supports it make since to just have it as an arg.